lib/ohai/plugins/windows/filesystem.rb

```diff
@@ -24,7 +24,7 @@
   # @see https://docs.microsoft.com/en-us/windows/desktop/SecProv/getconversionstatus-win32-encryptablevolume#parameters
   #
   CONVERSION_STATUS = %w{FullyDecrypted FullyEncrypted EncryptionInProgress
-                         DecryptionInProgress EncryptionPaused DecryptionPaused}.freeze
+                         DecryptionInProgress EncryptionPaused DecryptionPaused}.freeze unless defined?(CONVERSION_STATUS)
```

Comment:
1. I feel like this is a bandaid that hides a problem in `plugin.rb` (`NamedPlugin.plugin` is where this seems to be coming from).
2. You can change that to `||=` instead of `unless defined?`

Reply:
I updated this to improve our bandaid to use ||= everywhere